spec/api/documentation/events_api_spec.rb

```diff
@@ -655,7 +656,8 @@
           'maximum_polling_duration'   => new_plan.maximum_polling_duration,
           'create_instance_schema'     => new_plan.create_instance_schema,
           'update_instance_schema'     => new_plan.update_instance_schema,
-          'create_binding_schema'      => new_plan.create_binding_schema
+          'create_binding_schema'      => new_plan.create_binding_schema,
+          'maintenance_info' => new_plan.maintenance_info
```

Comment:
again, please keep trailing commas in hashes, please

Reply:
Same as above

Reply:
Fixed